### DISCUSSION TITLE
Can we Integrate with Browserstack

### DISCUSSION THREAD

**Discussion by rspriyamdixit** [Tallyb/cucumber-playwright#837] [Q&A]
Hi All, 
I'm Trying to integrate this repo with Browserstack using this doc
https://www.browserstack.com/docs/automate/playwright/getting-started/nodejs/integrate-your-tests-sdk

But its giving me below error
![error](https://github.com/Tallyb/cucumber-playwright/assets/85488390/a7f5395b-da73-417a-8a81-f8d4284c8020)

Can anyone help me here

Thanks in Advance

**Comment by sopansagorkar** [comment]
Make sure the BrowserStack dependency version in the repo and on the BrowserStack server match; usually, this error comes due to a mismatch.
